cosmo_tester/test_suites/image_based_tests/ssl_test.py

```diff
@@ -14,12 +14,12 @@
 #    * limitations under the License.
 
 from cosmo_tester.framework.examples.hello_world import centos_hello_world
-from cosmo_tester.framework.fixtures import bootstrap_based_manager
+from cosmo_tester.framework.fixtures import image_based_manager
 from cosmo_tester.framework.util import is_community
 from cloudify_rest_client.client import CloudifyClient
 from os.path import join
 
-manager = bootstrap_based_manager
+manager = image_based_manager
```

Comment:
could do the import line `..import image_based_manager as manager` which would make this particular commit a net - in lines! ;P

Reply:
I like it a little better that way. For starters, you'd have to add a #NOQA otherwise, and I think it's a tiny bit more explicit that way